libbpf-tools/trace_helpers.h

```diff
@@ -43,4 +43,34 @@ int bump_memlock_rlimit(void);
 
 bool is_kernel_module(const char *name);
 
+/*
+ * When we need for kprobe/kretprobe, please check out new fentry/fexit
+ * probes, as they provide better performance and usability. But in some
+ * situations we have to fallback to kprobe/kretprobe probes. This helper
+ * is used to detect those ones before attaching.
```

Comment:
"those ones" is ambiguous. Maybe "used to detect fentry/fexit support for the specified kernel function" ?